src/components/expression/expression-panel.js

```diff
@@ -113,8 +113,15 @@ genotet.ExpressionPanel.prototype.initPanel = function() {
     }.bind(this));
 
   // Input type update
+  var viewName = this.container.attr('id').replace('panel-view-', '');
```

Comment:
You can store a reference to the view object when the view is creating panel. In other words, you can remember for each tab/panel the view object it corresponding to. This can be potentially useful later as well.